### ISSUE TITLE
Story: Stevie Wonder

### ISSUE THREAD

**Issue by dotto-botto** [Allen-ISD-Computer-Science/REPO_orangetrading.com_2024#24]
As a blind person, I should be able to read the text on the website using a screen reader.
